### DISCUSSION TITLE
安装了 头像没啥反应 要设置什么吗 ......小白一枚

### DISCUSSION THREAD

**Discussion by wangqi12323** [ddd354/JAVnfoMoviesImporter.bundle#8] [Q&A]
![屏幕截图 2022-04-13 230738](https://user-images.githubusercontent.com/91153346/163212457-9cd58664-7b96-486a-a27d-7f615f580fc4.png)
![屏幕截图 2022-04-13 230833](https://user-images.githubusercontent.com/91153346/163212475-061b3e58-d185-4ef9-8413-a036a1e84e56.png)
![屏幕截图 2022-04-13 231003](https://user-images.githubusercontent.com/91153346/163212482-b3e1fb96-7322-41f4-8e61-417fdfbae527.png)
大佬 帮我看看有没有设置错0.0

**Comment by ddd354** [comment]
一般都先考虑网络的问题 plex服务器用全局代理试试
